README.md

```diff
@@ -153,15 +153,15 @@ From SwaggerHub Explore, navigate to your browser development tools, locate the
                 |_|
 ```
 **Description:**
-  > Import SwaggerHub Explore spaces from a file
+  > Import Postman collections from a file
```

Comment:
```suggestion
  > Import Postman collections (v2.1) from a file
```

Reply:
Thanks @JosephBJoyce 

One minor suggestion above, and also could you update the text description in line 29 to state: "Import a Postman collection v2.1 into SwaggerHub Explore"

Reply:
Done and done!